### ISSUE TITLE
Show exact matches first when doing search

### ISSUE THREAD

**Issue by AMDmi3** [repology/repology#351]

**Comment by jrmarino** [comment]
yes please.  Many times the exact match wasn't part of the first 50 returned, so it didn't show at all.

I might also suggest putting higher priority on first part matches, e.g. "box" would list "boxcutter" before "abox"
